### ISSUE TITLE
Add a CODE_EXTENSIONS_PATH similar to CODE_TEST_PATH?

### ISSUE THREAD

**Issue by vazexqi** [Microsoft/vscode-extension-vscode#70]
I would like to write tests that exercise a collection of extensions. I can do so interactively using [Launch
Tests Configurations](https://code.visualstudio.com/docs/extensions/testing-extensions#_launch-tests-configuration) using `--extensionDevelopmentPath`. 

I would like to do the same for CI tests. But the [test](https://github.com/Microsoft/vscode-extension-vscode/blob/master/bin/test#L49) script is hardcoded to use `process.cwd()`

```
 var args = [
        testsWorkspace,
        '--extensionDevelopmentPath=' + process.cwd(),
        '--extensionTestsPath=' + testsFolder
    **];**
```

Could we also have a CODE_EXTENSIONS_PATH variable? The code to do this is quite simple and I can make a PR for this if there are no objections.

**Comment by bpasero** [comment]
Makes sense to me, PR welcome 👍

**Comment by bpasero** [comment]
Published as 1.1.2
